terraspace/app/stacks/shared/auth.tf

```diff
@@ -122,3 +122,90 @@ EOF
     create_before_destroy = true
   }
 }
+
+
+resource "aws_iam_policy" "dynamodb_v1_blocks_policy" {
+  name        = "dynamodb-${var.v1_blocks_table.name}-policy"
```

Comment:
Remind me, how are we ensuring the name is unique across environments/regions ?

Reply:
v1_blocks_table = {
  name     = "<%= expansion(':ENV') %>-ep-v1-blocks"